### ISSUE TITLE
The customVariables feature is not working properly

### ISSUE THREAD

**Issue by lvarado** [nuxt-community/vuetify-module#173]
**Module version**
1.8.5

**Describe the bug**
Disclaimer: I'm new to Vuetify and somewhat new to Nuxt, so please forgive any ignorance.

I tried to override default variables using `customVariables` as outlined in the docs. However, these seem to have no effect. The docs say to enable `treeShake`, which I tried in both `nuxt.config.js` and `vuetify.config.js` with no success. I also tried moving `customVariables` to `vuetify.config.js` but that didn't work either.

**To Reproduce**
CSB fork: https://codesandbox.io/s/nuxtjs-vuetify-p00e2

1. Check `nuxt.config.js`; note that `treeShake` is set to `true` and `customVariables` is set to `["~/assets/variables.scss"]`.
2. Check `~/assets/variables.scss` to see the following:
```scss
$font-size-root: 24px;
$body-font-family: 'Georgia', serif;
```
3. Look at the preview and notice the font size and family remain default values

**Expected behavior**
I expect the base font size and body font family to use the values set in `~/assets/variables.scss`.

**Screenshots**
<img width="778" alt="Screen Shot 2019-09-24 at 10 10 34 AM" src="https://user-images.githubusercontent.com/32886847/65524541-9877a180-deb3-11e9-819d-dfd121366499.png">

**Comment by lvarado** [comment]
NB: the [unit test](https://github.com/nuxt-community/vuetify-module/blob/master/test/module.test.ts#L94-L100) does pass, but I noticed it only checks for existence of the `@import` line. If this bug is indeed valid instead of user error, perhaps it would be good for the test to check compiled CSS for a custom value.

**Comment by kevinmarrec** [comment]
@lvarado Didn't change anything and opened your CSB, it works for me ...
![image](https://user-images.githubusercontent.com/25272043/65601867-39915580-dfa3-11e9-8db2-58e26c518423.png)

**Comment by lvarado** [comment]
I swear my screenshot is legit! But it's working fine for me now too...

My only theory is maybe something was buggy with version `1.8.3`. I manually updated the version number in `package.json` on CSB from  `1.8.3` to `1.8.5` but at the time I took the screenshot it may have still been using the cached older version, and the cache was deleted overnight. I did restart the sandbox but I'm not sure if the module actually got updated.

The local version of my project was still broken as of this morning until I ran `yarn upgrade --latest`, at which point the variables started working. Unfortunately I did not check `yarn.lock` to see if I was on that older version before doing this, so it could be an update to this module or another dependency that resolved the issue.

Either way, apologies for the wild goose chase and thank you for your quick reply @kevinmarrec. I'll go ahead and close this issue.
